pkg/cmd/publish/publish.go

```diff
@@ -185,16 +186,42 @@ func (cmd *publishCmd) run(f *cmdutil.Factory, info *publishInfo, options *contr
 		}
 	}
 
-	fmt.Fprintf(cmd.f.IOStreams.Out, "\nYour Domain name: %s\n", domain.GetDomainName())
+	var domainReturnedName []string
+	domainReturnedName = append(domainReturnedName, domain.GetDomainName())
+
+	fmt.Fprintf(cmd.f.IOStreams.Out, "\nYour Domain name: %s\n", domainReturnedName[0])
 
 	err = utils.WriteAzionJsonContent(conf)
 	if err != nil {
 		return err
 	}
 
+	if conf.RtPurge.PurgeOnPublish {
+		err = cmd.purgeDomain(f, domainReturnedName)
+		if err != nil {
+			return err
+		}
+	}
+
 	return nil
 }
 
+func (cmd *publishCmd) purgeDomain(f *cmdutil.Factory, domainReturnedName []string) error {
+	ctx := context.Background()
+	clipurge := apipurge.NewClient(f.HttpClient, f.Config.GetString("api_url"), f.Config.GetString("token"))
+	pg, err := clipurge.Purge(ctx, &domainReturnedName)
+	if err != nil {
+		return err
+	}
+
+	if pg.StatusCode == 201 {
+		fmt.Fprintf(cmd.f.IOStreams.Out, "\nDomain cache was purged\n")
+		return nil
+	} else {
+		return fmt.Errorf("%s: %w", errmsg.ErrorPurgeDomainCache, err)
+	}
+}
```

Comment:
You can avoid the else here
```suggestion
	if pg.StatusCode != 201 {
	        return fmt.Errorf("%s: %w", errmsg.ErrorPurgeDomainCache, err)
	}
        fmt.Fprintf(cmd.f.IOStreams.Out, "\nDomain cache was purged\n")
        return nil
}
```
https://github.com/uber-go/guide/blob/master/style.md#unnecessary-else

Reply:
done